deepspeed/profiling/gpu_profiler/gpu_profiler.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from torch.profiler import profile, record_function, ProfilerActivity
+class GPUProfiler(object):
+    def __init__(self, model):
+        self.model = Model()
+
+    def start_gpu_profiler(inputs):
+        with profile(activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA], record_shapes=True,profile_memory=True) as prof:
```

Comment:
Why does this include `ProfilerActivity.CPU`?

Reply:
Sorry I added by mistake! , I remove ProfilerActivity.CPU